src/main/webapp/app/assessment/unreferenced-feedback-detail/unreferenced-feedback-detail.component.html

```diff
@@ -10,7 +10,7 @@
             <button
                 jhiDeleteButton
                 [buttonSize]="ButtonSize.SMALL"
-                [translateValues]="{ text: feedback.detailText }"
+                [translateValues]="{ text: Feedback.hasContent(feedback) ? ' \'' + Feedback.getContent(feedback) + '\'' : '' }"
```

Comment:
Please avoid calling methods from your html template. Instead, you should calculate the desired values beforehand and assign them to a variable.

Reply:
Best I can do is put the value in a `get` attribute (see new commit). The user can still change the feedback text and the grading instruction.